README.md

```diff
@@ -560,6 +560,85 @@ Will result in help output like:
 --lang value, -l value  Language for the greeting (default: "english")
 ```
 
+#### Required Flags
+
+You can make a flag required by setting the `Required` field to `true`. If an end-user
+fails to provide a required flag, they will be shown a default error message, or a 
+custom message if one is defined. To define a custom error message for a required flag,
+set the `RequiredFlagErr` field equal to a `cli.FlagErr` struct with a `Custom` field of `true`
+and a `Message` field containing the custom error message. Default and custom error messages 
+can be mixed with default messages displayed before custom messages.
+
+For example this:
+
+```go
+package main
+
+import (
+  "log"
+  "os"
+  "strings"
+
+  "github.com/urfave/cli"
+)
+
+func main() {
+  app := cli.NewApp()
+
+  app.Flags = []cli.Flag {
+    cli.StringFlag{
+      Name: "lang",
+      Value: "english",
+      Usage: "language for the greeting",
+      Required: true, 
+      RequiredFlagErr: cli.FlagErr{
+        Custom: true, 
+        Message: `There's a problem: "lang" is a required flag` ,
```

Comment:
⭐⭐ _(seriously consider)_ In my experience the most common use case for required flags is when a user must pick from a set of choices. Having the example reflect that would likely help users make good decisions, so something like

> There's a problem: "lang" is required, and must be EN, ES, or RU

Reply:
👋 I've had my focus shift to other things, and don't have the capacity to provide much more guidance here. My recommendation is that PR be closed. I believe that we will return here in the future, though! ✨

Reply:
Me too. Just went through a major release at work so haven't had any free time. Revisiting some time in the future sounds good. Thanks!